src/index.js

```diff
@@ -200,16 +209,16 @@ class Dayjs {
         step = C.MILLISECONDS_A_SECOND
     }
     const nextTimeStamp = this.valueOf() + (number * step)
-    return new Dayjs(nextTimeStamp)
+    return dayjs(nextTimeStamp)
   }
 
   subtract(number, string) {
     return this.add(number * -1, string)
   }
 
-  format(formatStr = 'YYYY-MM-DDTHH:mm:ssZ') {
-    const weeks = 'Sunday.Monday.Tuesday.Wednesday.Thursday.Friday.Saturday'.split('.')
-    const months = 'January.February.March.April.May.June.July.August.September.October.November.December'.split('.')
+  format(formatStr = this.$format, L = {}) {
```

Comment:
I think this could be improved by using:

```js
format(formatStr = this.$format, L = this.$L)
```

Then I think you can get rid of next lines short circuits `||`. What do you think?

Reply:
Really depends, because you may do `format.('...', { WD: [0,1,2,3,4,5,6] })` to override the week days locale, but leave `MONTHS` untouched with the default instance locale. You're not setting a locale, just providing overrides for certain locales.